AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -2445,6 +2445,8 @@ izlemac75.net##body > div[style*="width:100%;text-align:center"]
 izlemac75.net##body > div#reload
 !
 bessarabiainform.com##.news-bar
+full-count.jp##.erase_space-ad
+full-count.jp#?#section > div.c-heading:has(> p.c-heading__en:contains(RECOMMEND))
```

Comment:
Why not to `widgets.txt`? You may create sub-section with the corresponding comment in the end of `! Suggestion widgets`(with url blocking rule)